apps/alerting/historian/pkg/app/notification/lokireader.go

```diff
@@ -56,9 +58,13 @@ func NewLokiReader(cfg lokiclient.LokiConfig, reg prometheus.Registerer, logger
 		Buckets:   instrument.DefBuckets,
 	}, instrument.HistogramCollectorBuckets))
 
+	requester := &http.Client{
+		Transport: cfg.Transport,
```

Comment:
Can cfg.Transport be nil, do we need a fallback to the default transport?

Reply:
Yeah default is used if it's nil